tests/schema_registry/test_api_client.py

```diff
@@ -376,3 +377,13 @@ def test_schema_equivilence(load_avsc):
     assert schema == schema2
     assert schema_str1.__eq__(schema_str2)
     assert schema_str1 == schema_str2
+
+def test_timeout_exception_thrown_when_unreasonably_low_timeout_set(mock_schema_registry):
+    conf = {'url': TEST_URL,
+        'request.timeout' : 0.01   
```

Comment:
if TEST_URL is a valid URL I don't think a request timeout of 10ms is sufficienctly reliable to be exceeded.
Instead suggest creating a TCP listener that you connect to.